v19.1/encryption.md

```diff
@@ -21,8 +21,7 @@ Encryption at Rest provides transparent encryption of a node's data on the local
 If you encounter a bug, please [file an issue](file-an-issue.html).
 {{site.data.alerts.end}}
 
-Encryption at Rest allows encryption of all files on disk using [AES](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard) in [counter mode](https://en.wikipedia.org/wiki/Block_cipher_mode_of_operation#Counter_(CTR)), with all key
-sizes allowed.
+Encryption at Rest allows encryption of all files on disk using [AES](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard) in [counter mode](https://en.wikipedia.org/wiki/Block_cipher_mode_of_operation#Counter_(CTR)), with all AES-Standard key sizes allowed.
```

Comment:
Encryption at rest seems like it belongs in its own page

Reply:
I consolidated the encryption docs as part of the security project I did last quarter. Making a separate doc for encryption at rest would warrant an independent discussion. Making a note of it.